pkg/sql/parser/sql.y

```diff
@@ -5137,9 +5137,13 @@ show_zone_stmt:
     $$.val = &tree.ShowZoneConfig{}
   }
 
-for_or_from:
+for_implicit_alias_from:
```

Comment:
maybe `for_with_implicit_from_alias` which is more descriptive when reading the BNF diagrams

Reply:
Changed.